src/ILCompiler/src/Program.cs

```diff
@@ -343,6 +343,9 @@ private int Run(string[] args)
 
             typeSystemContext.InputFilePaths = inputFilePaths;
             typeSystemContext.ReferenceFilePaths = _referenceFilePaths;
+            if (!typeSystemContext.InputFilePaths.ContainsKey(_systemModuleName)
+                && !typeSystemContext.ReferenceFilePaths.ContainsKey(_systemModuleName))
+                throw new CommandLineException($"System module {_systemModuleName} does not exists. Make sure that you specify --systemmodulename");
```

Comment:
Could you make that --systemmodule?

Reply:
Done

Reply:
> Make sure that you specify --systemmodule

...or reference System.Private.CoreLib assembly?